Makefile

```diff
@@ -970,13 +970,6 @@ e2e-aws: $(TEST_LOG_DIR) ensure-gotestsum
 .PHONY: lint
 lint: lint-api lint-go lint-kube-agent-updater lint-tools lint-protos lint-no-actions
 
-#
-# Lints everything but Go sources.
-# Similar to lint.
-#
-.PHONY: lint-no-actions
-lint-no-actions: lint-sh lint-helm lint-license lint-rust
```

Comment:
This target no longer necessary since the lint workflow runs each of these targets individually depending on what changed.